src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.cs

```diff
@@ -1259,7 +1260,7 @@ public override string Text
                     }
                     else
                     {
-                        return FilterItemOnProperty(SelectedItem).ToString();       //heinous.
+                        return FilterItemOnProperty(SelectedItem)!.ToString()!;       //heinous.
```

Comment:
This makes me very uneasy.... @dotnet/dotnet-winforms what do you think of this?
```suggestion
                        return FilterItemOnProperty(SelectedItem)?.ToString() ?? base.Text;
```